pkg/build/nodeimage/defaults.go

```diff
@@ -20,7 +20,7 @@ package nodeimage
 const DefaultImage = "kindest/node:latest"
 
 // DefaultBaseImage is the default base image used
-const DefaultBaseImage = "kindest/base:v20201130-23777eca"
+const DefaultBaseImage = "kindest/base:v20201204-49cad832"
```

Comment:
ping @BenTheElder , however, this doesn't solve the problem

Reply:
where did you pull that from? note: random images in this repo are not supported.
I dont think we missed any bumps merging.

Reply:
pulled from here https://hub.docker.com/r/kindest/base/tags?page=1&ordering=last_updated , it says updated 11 days ago